UnicornEngine/video/include/unicorn/video/vulkan/VkTexture.hpp

```diff
@@ -27,33 +27,38 @@ namespace vulkan
 class VkTexture
 {
 public:
+    /**
+     * @brief Default constructor for VkTexture
+     * @param device device which allocate from
+     */
     VkTexture(vk::Device device);
 
     /**
-     * @brief 
+     * @brief Getter for descriptor image info
```

Comment:
Please describe _what it does_, not _what it is_. `Returns decriptior image info`
For simple cases like getters you can write oneline documentation
`/** Returns descriptor image info */`